modules/standard/Math.chpl

```diff
@@ -178,6 +178,7 @@ Constant and Function Definitions for Math
 module Math {
   private import HaltWrappers;
   private use CTypes;
+  private import BitOps;
```

Comment:
PR #23884 undid other changes in this file but not this one. Perhaps that is the reason we are still seeing nightly failures.